src/box/session_settings.c

```diff
@@ -343,7 +343,8 @@ session_settings_space_execute_update(struct space *space, struct txn *txn,
 	const char *new_key, *key = request->key;
 	uint32_t new_size, new_key_len, key_len = mp_decode_array(&key);
 	if (key_len == 0) {
-		diag_set(ClientError, ER_EXACT_MATCH, 1, 0);
+		diag_set(ClientError, ER_EXACT_MATCH_DETAILED,
+			 space_name(space), pk_def->name, 2, 0);
```

Comment:
Why did you change `1` to `2`? There must be only one part, not two. It's a pity that we don't have a test for it. Perhaps we should add one.

Reply:
Fixed.